src/popup/locales/en.json

```diff
@@ -1317,6 +1317,7 @@
       "version": "Contract version",
       "currentNonce": "Current nonce",
       "authorizedAccount": "Authorized accounts",
+      "authorizedSigners": "Authorized signers",
```

Comment:
On my current branch I added this to the "multisig" entry (line 42).
```
    "multisigVault": "Multisig vault",
    "authorizedSigners": "Authorized signers",
    "needForApproval": "Needed for approval"
```
Can you do the same to avoid duplication?